src/query/formats/src/field_decoder/csv.rs

```diff
@@ -48,7 +47,7 @@ impl FieldDecoderCSV {
                 true_bytes: TRUE_BYTES_LOWER.as_bytes().to_vec(),
                 false_bytes: FALSE_BYTES_LOWER.as_bytes().to_vec(),
                 null_bytes: NULL_BYTES_ESCAPE.as_bytes().to_vec(),
-                nan_bytes: NAN_BYTES_LOWER.as_bytes().to_vec(),
+                nan_bytes: options.stage.nan_display.as_bytes().to_vec(),
```

Comment:
other formats need to use stage.nan_display later.
we should change them now.
so we can tests it`s default values properly handled (should not change to NaN or "") after this pr.